tests/fvt/conftest.py

```
@@ -24,7 +24,9 @@ def custom_context():
 def copied_config_files():
     base_dir = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))  # loopy/
     config_path = os.path.join(base_dir, "config.yaml")
-    default_vars_path = os.path.join(base_dir, "commons", "default-variables.yaml")
+    default_vars_path = os.path.join(
```

Ruff (0.11.9)

Local variable `default_vars_path` is assigned to but never used

Remove assignment to unused variable `default_vars_path`

(F841)